yt_dlp/extractor/ixigua.py

```diff
@@ -0,0 +1,66 @@
+from .common import InfoExtractor
+from ..utils import (
+    get_element_by_id,
+    js_to_json,
+    traverse_obj,
+    ExtractorError,
+)
+import base64
+
+
+class IxiguaIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:\w+\.)?ixigua\.com/(?:video/)?(?P<id>[0-9]+).+'
+    _TEST = {
+        'url': 'https://www.ixigua.com/6996881461559165471',
+        'info_dict': {
+            'id': 'v0d004g10000c4d1t7jc77ub4g3o88b0',
+            'ext': 'unknown_video',
+            'title': '盲目涉水风险大，亲身示范高水位行车注意事项',
+            'description': '本期《懂车帝评测》，我们将尝试验证一个夏日大家可能会遇到的关键性问题：如果突发暴雨，我们不得不涉水行车，如何做才能更好保障生命安全。',
+            'tag' : 'video_car'
+            # thumbnail url keep changing
```

Comment:
use regex

Reply:
Fixed in 01da9de4aebca2d9c3dccf8c855d8cbc422386d7